virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,47 @@
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
-
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
-
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+You must create a Linux bridge network attachment definition (NAD) in order to connect to additional networks.
 
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
+To attach a virtual machine to an additional network:
 
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
+. Create a Linux bridge node network configuration policy.
+. Create a Linux bridge network attachment definition.
+. Configure the virtual machine, enabling the virtual machine to recognize the network attachment definition.
 
 For more information about scheduling, interface types, and other node networking activities,
 see the xref:../../../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#k8s-nmstate-updating-node-network-config[node networking]
 section.
 
-[id="virt-creating-network-attachment-definition"]
-== Creating a network attachment definition
+[id="virt-connecting-network-through-nad"]
+== Connecting to the network through the network attachment definition
+
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+3]
+
+[id="virt-creating-linux-bridge-network-attachment-definition"]
+== Creating a Linux bridge network attachment definition
 
 [WARNING]
 ====
-Configuring ipam in a network attachment definition for virtual machines is not supported.
+Configuring IP address management (IPAM) in a network attachment definition for virtual machines is not supported.
 ====
 
-include::modules/virt-creating-bridge-nad-web.adoc[leveloffset=+2]
-
-include::modules/virt-creating-bridge-nad-cli.adoc[leveloffset=+2]
-
-[id="virt-attaching-vm-to-additional-network"]
-== Attaching the virtual machine to the additional network
-
-include::modules/virt-vm-creating-nic-web.adoc[leveloffset=+2]
-
-include::modules/virt-networking-wizard-fields-web.adoc[leveloffset=+2]
+include::modules/virt-creating-bridge-nad-web.adoc[leveloffset=+3]
 
-include::modules/virt-attaching-vm-secondary-network-cli.adoc[leveloffset=+2]
+include::modules/virt-creating-bridge-nad-cli.adoc[leveloffset=+3]
 
-[id="{context}_additional-resources"]
-[role="_additional-resources"]
-== Additional resources
+[id="virt-configuring-vm-for-additional-network"]
+== Configuring the virtual machine for an additional network
```

Comment:
For the sake of consistency, you might want to change "additional network" to "Linux bridge network", since that it what it is called in the rest of the assembly.

Reply:
Done, changed ID as well.